onnx/proto_utils.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+#include <google/protobuf/io/coded_stream.h>
+#include <google/protobuf/io/zero_copy_stream_impl_lite.h>
+#include <pybind11/pybind11.h>
```

Comment:
I'd rename proto_utils.h to py_utils.h; the Python dependency is more interesting than the protobuf one, I think. Over time I'm sure we'll find plenty of things that we'll want to put in a "proto_utils.h" file, without pulling in pybind.

Reply:
Good point, we should separate protobuf && python. I will add another file py_utils.h which will include this proto_utils.h.